src/Tracking.php

```diff
@@ -128,6 +143,39 @@ public function trackByReference($referenceNumber, $requestOption = 'activity')
     }
 
     /**
+     * Set shipper number
+     *
+     * @param string $shipperNumber
+     *
+     */
+    public function setShipperNumber($shipperNumber)
+    {
+        $this->shipperNumber = $shipperNumber;
+    }
+
+    /**
+     * Set begin date
+     *
+     * @param string $beginDate
+     *
+     */
+    public function setBeginDate($beginDate)
+    {
+        $this->beginDate = $beginDate;
+    }
+
+    /**
+     * Set end date
+     *
+     * @param string $endDate
+     *
+     */
+    public function setEndDate($endDate)
```

Comment:
Is it an idea to use DateTime objects in here? That prevents people from inserting a wrong format.